cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerLoggingConfigurationDiff.java

```diff
@@ -96,6 +95,80 @@ public int getDiffSize() {
         return updatedCE;
     }
 
+    private static Map<String, String> readLog4jConfig(String config) {
+
+        Map<String, String> parsed = new LinkedHashMap<>();
+        Map<String, String> env = new HashMap<>();
+        BufferedReader reader = new BufferedReader(new StringReader(config));
+        String line;
+        try {
+            while ((line = reader.readLine()) != null) {
+                // skip comments
+                if (line.startsWith("#")) continue;
+
+                // ignore empty lines
+                line = line.trim();
+                if (line.length() == 0) continue;
+
+                // everything that does not start with 'log4j.' is a variable definition
+                if (!line.startsWith("log4j.")) {
+                    int p = line.indexOf("=");
```

Comment:
What about spaces before `=`?

Reply:
trim() takes care of those, or do you have something else in mind?

Reply:
Ah, yes, you are right

Reply:
Can we have a better variable name than `p`?

Reply:
Can we have a better variable name than p?

Reply:
Hmm, looks like I missed this one.